site/docs/v1/tech/apis/scim/_scim-response-codes.adoc

```diff
@@ -14,21 +14,25 @@ ifdef::delete[]
 endif::[]
 
 |400
-|The request was invalid and/or malformed. The response will contain an link:/docs/v1/tech/apis/scim[SCIM Errors] JSON Object with the specific errors. This status will also be returned if a paid FusionAuth license is required and is not present.
+|The request was invalid and/or malformed. The response will contain a link:/docs/v1/tech/apis/scim[SCIM Error] JSON Object with the specific errors. This status will also be returned if a paid FusionAuth license is required and is not present.
 
+ifndef::unauthenticated[]
 |401
 |You did not supply a valid JWT in your Authorization header. The response will be empty. Ensure you've correctly set up Entities and performed a link:/docs/v1/tech/apis/authentication#client-credentials[Client Credentials] grant.
+endif::[]
 
-ifndef::create[]
+ifndef::no_missing[]
 |404
 |The object doesn't exist. The response will be empty.
 endif::[]
 
 |500
 |There was an internal error. A stack trace is provided and logged in the FusionAuth log files. The response will be empty.
 
+ifndef::no_search[]
```

Comment:
`never_search_error` seems to be more standard

Reply:
done